lark/lark.py

```diff
@@ -115,6 +119,8 @@ class LarkOptions(Serialize):
         'edit_terminals': None,
         'g_regex_flags': 0,
         'use_bytes': False,
+        'import_sources': [],
+        'source': None,
```

Comment:
`source` is a little vague in this context. Maybe call it `source_name` or `source_path`.

Reply:
Yes, true

Reply:
Would you be ok with also renaming `Lark.source`? It is as vague. (Also I would rename `Lark.grammar_source` to `Lark.source_code`)

Reply:
In principle I agree, but it would break compatibility. You can open an issue for this for v1.0

Fwiw, at least when you read the existing `source` property, you can understand what it is from its value.

Reply:
I would of course add backwards compatibility via properties.

Reply:
Okay, I suppose there's no harm